GoogleForCreators/web-stories-wp
# Text Sets: make use of custom heading levels

## Context

Updates `cover` text sets to ensure "primary" title gets assigned an `H1` tag


## Summary

Updates cover text sets



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1). Create new story
2) Add each `cover` text set to a page
3) Ensure the primary title is showing as an `H1` (Style -> Accessibility -> heading level)

<img width="370" alt="Screen Shot 2022-07-27 at 4 43 36 PM" src="https://user-images.githubusercontent.com/62242/181368908-933cd774-01ab-4ad8-973a-2b992df3dfe1.png">


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?

No



### Does this PR have a legal-related impact?

No



## Checklist




---



Fixes #10724
